elide-model-config/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/compile/ElideDynamicEntityCompiler.java

```diff
@@ -271,4 +277,16 @@ public static String getStaticModelClassImport(String modelName, String modelVer
         ModelMapValue value = STATIC_MODEL_DETAILS.get(new ModelMapKey(modelName, modelVersion));
         return value != null ? value.getClassImport() : defaultValue;
     }
+
+    /**
+     * Checks if field exists in referenced static model.
+     * @param modelName model name.
+     * @param modelVersion model version.
+     * @param fieldName field name to check.
+     * @return true if model has the field defined else false
+     */
+    public static boolean staticModelContainsField(String modelName, String modelVersion, String fieldName) {
```

Comment:
Is there a reason these functions need to be static?

Reply:
defined as static so that it can be used for both DynamicConfigValidator and HandlebarsHydrator easily.